core/templates/dev/head/pages/about/About.js

```diff
@@ -19,6 +19,45 @@
 oppia.controller('About', [
   '$scope', 'UrlInterpolationService',
   function($scope, UrlInterpolationService) {
+    var activeTabClass = 'oppia-about-tabs-active';
+    var hash = window.location.hash.slice(1);
+    var visibleContent = 'oppia-about-visible-content';
+
+    var activateTab = function(tabName) {
+      $("a[id='" + tabName + "']").parent().addClass(
+        activeTabClass
+      ).siblings().removeClass(activeTabClass);
+      $('.' + tabName).addClass(visibleContent).siblings().removeClass(
+        visibleContent);
+    };
+
+    if (hash === 'foundation' || hash === 'license') {
+      activateTab('foundation');
```

Comment:
Better to pull these strings out into controller-level constants like `$scope.TAB_ID_FOUNDATION`, and use the named constants everywhere (including in the HTML) instead of the raw strings. That guards against future typos and also allows the name to be changed in a single place.